client/src/components/FrequentlyAskedQuestions.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Stack, Text, StackItem, Icon, Link } from '@fluentui/react';
+import {
+  btnStackStyles,
+  font16pxStyle,
+  linkStyles,
+  newWindowIconWrapper,
+  textDecorationNone
+} from '../styles/FrequentlyAskedQuestions.styles';
+
+export const FrequentlyAskedQuestions: React.VoidFunctionComponent = () => {
+  return (
+    <>
+      <Text styles={font16pxStyle}>Frequently asked questions</Text>
+      <Stack styles={btnStackStyles}>
+        <Link
+          tabIndex={0}
+          data-testid="bookingsSetupLink"
```

Comment:
This will prolly be removed when merged with main since it was removed in mine :) .. But just double-check